src/content/1.7/development/architecture/domain/services.md

```diff
@@ -1,44 +1,59 @@
 ---
 title: Domain services
+
 weight: 20
 ---
 
 # Domain services
 
-As mentioned in [CQRS principles](cqrs.md), a single command is just a port, a representation of a single use case, therefore the command handler should not contain any domain logic by itself, but should be orchestrating other services to fulfill the use case instead.
-These services are responsible for logic such as:
+As mentioned in [CQRS principles](../cqrs), a `Command` is a representation of a single domain use case. To
+prevent `Command Handlers` from duplicating the code and depending on each other, it is usually best to **avoid keeping
+the actual logic in the `Command Handler`** itself, but delegate it to the `Domain Services`. These services are responsible for
+logic such as:
```

Comment:
Avoid breaking paragraphs, use soft wraps in your editor as necessary
```suggestion
As mentioned in [CQRS principles]({{< relref "/cqrs" >}}), a `Command` is a representation of a single domain use case. To prevent `Command Handlers` from duplicating code and depending on each other, it is usually best to **avoid implementing the actual logic in the `Command Handler`** itself, and delegate it to `Domain Services` instead. These services are responsible for logic such as:
```